config/locales/en.yml

```diff
@@ -353,6 +354,9 @@ en:
ody">We use Google Analytics software to collect anonymised information about how you use RODA. We do this to help make sure the site is meeting the needs of its users and to help us make improvements to the site.</p> <p class="govuk-body">We do not allow Google to use or share the data about how you use this site.</p> <p class="govuk-body">Google Analytics stores information about:</p> <ul class="govuk-list govuk-list--bullet"> <li>how you got to the site</li> <li>the pages you visit on RODA and how long you spend on them</li> <li>what you click on while you’re visiting the site</li> </ul> <p class="govuk-body">Google Analytics sets the following cookies:</p> <table class="govuk-table"> <thead class="govuk-table__head"> <tr class="govuk-table__row"> <th scope="col" class="govuk-table__header">Name</th> <th scope="col" class="govuk-table__header">Purpose</th> <th scope="col" class="govuk-table__header">Expires</th> </tr> </thead> <tbody class="govuk-table__body"> <tr class="govuk-table__row"> <td class="govuk-table__cell">_ga</td> <td class="govuk-table__cell">These help us count how many people visit RODA by tracking if you’ve visited before</td> <td class="govuk-table__cell">2 years</td> </tr> <tr class="govuk-table__row"> <td class="govuk-table__cell">_gid</td> <td class="govuk-table__cell">These help us count how many people visit RODA by tracking if you’ve visited before</td> <td class="govuk-table__cell">24 hours</td> </tr> <tr class="govuk-table__row"> <td class="govuk-table__cell">_gat_UA-151431879-2</td> <td class="govuk-table__cell">Used to  control the number of requests RODA makes to Google Analytics</td> <td class="govuk-table__cell">1 minute</td> </tr> </tbody> </table> <h2 class="govuk-heading-m">Strictly necessary cookies</h2> <p class="govuk-body">These cookies do not store your personal data but are required for the service to operate.</p> <table class="govuk-table"> <thead class="govuk-table__head"> <tr class="govuk-table__row"> <th scope="col" class="govuk-table__header">Name</th> <th scope="col" class="govuk-table__header">Purpose</th> <th scope="col" class="govuk-table__header">Expires</th> </tr> </thead> <tbody class="govuk-table__body"> <tr class="govuk-table__row"> <td class="govuk-table__cell">_roda_session</td> <td class="govuk-table__cell">Identifies your session in RODA</td> <td class="govuk-table__cell">When you close your browser or 24 hours</td> </tr> </tbody> </table> <h2 class="govuk-heading-m">Gathering your feedback</h2> <p class="govuk-body">We use Google Forms software to collect information about what you thought of the RODA service. We do this to help make sure the site is meeting the needs of its users and to help us make improvements.</p> <p class="govuk-body">For more information on Google‘s cookies, please see the <a href="https://policies.google.com/technologies/cookies?hl=en-UK" class="govuk-link">Google Privacy and Terms</a> site.</p> <p class="govuk-body published-dates">Last updated 21 May 2020</p>
```

Comment:
I'm not sure how a HTML blob is easier to maintain than a structured view to be honest. I would assume that it is easier to edit HTML when looking at it with its structure and indentation? I realise this file would be changing less frequently but we work with the en.yml file a lot but would be less likely to open a cookies.html. 

As a similar static page our privacy policy content has already been set up as conventional view file rather than being managed as a blob so this new approach introduces an inconsistency. 

Have I missed a benefit here?

Reply:
I've only just realised that the privacy policy isn't internationalised 😱 

You haven't missed a benefit, I will make the cookie statement like the other public pages (only internationalised)